l10n_it_fiscalcode/migrations/10.0.1.0.1/post-migrate.py

```diff
@@ -14,5 +14,6 @@ def migrate(cr, version):
         env = api.Environment(cr, SUPERUSER_ID, {})
         partners = env['res.partner'].search(
             [('is_company', '=', True)])
-        for partner in partners:
-            partner._commercial_sync_to_children()
+        for idx, partner in enumerate(partners):
```

Comment:
`partners` è già un recordset, a cosa ti serve utilizzare `enumerate`?

Reply:
Scusa, lo avevo utilizzato per dei print e lo avevo dimenticato.
Non serve in effetti, l'ho tolto.